primefaces/src/main/java/org/primefaces/component/selectoneradio/SelectOneRadioRenderer.java

```diff
@@ -97,8 +97,11 @@ protected void encodeResponsiveLayout(FacesContext context, SelectOneRadio radio
         String clientId = radio.getClientId(context);
         List<SelectItem> selectItems = getSelectItems(context, radio);
         String style = radio.getStyle();
+        boolean flex = ComponentUtils.isFlex(context, radio);
+        if (flex) {
+            layout = "responsive";
+        }
```

Comment:
Is this something to mention in the migration guide?

Reply:
Good point I just added a note to migration guide